docker-compose.yml

```diff
@@ -19,6 +19,11 @@ services:
     image: ubautograding/devtools
     volumes:
       - ./devU-api/config:/config
+  shared-modules:
+    build:
+      context: devu-shared/
+    volumes:
+      - ./devU-api:/app
```

Comment:
I think I like this solution since we only compile shared once. We'll need to get this in the client as well